README.md

```diff
@@ -64,12 +50,9 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Microverse
 
-## 📝 License
 
-This project is [MIT](./LICENSE) licensed.
+## 📝 License
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT](./MIT.md) licensed.
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/195865110-c8cb8232-4c91-4866-8e11-ea4599385d89.png)
**[Optional]**
- [ ] Kindly add your name, current year and app-type in your MIT.md file. This is the license file of your project so kindly take it seriously :blush: